src/get-nohoist.js

```diff
@@ -24,7 +24,8 @@ module.exports = function getNohoist(params = {}) {
     : getWorkspaces({ cwd });
   const monorepoRoot = getMonorepoRoot({ cwd });
   const packageJson = require(path.join(monorepoRoot, "package.json"));
-  const nohoistGlobs = packageJson.workspaces.nohoist || [];
+  const nohoistGlobs = [...(packageJson.workspaces.nohoist || []), 
+    ...(require(path.join(cwd, "package.json")).workspaces?.nohoist || [])];
```

Comment:
We don't support the `?` operator yet, since we want be backward compatible with older versions of node. Also, could you add a comment to explain what/why we're doing this?
Thanks!